### PR TITLE
🐳 Automate publisher ci

### PR DESCRIPTION
**Pull request by liberaliscomputing**
This PR automates the publisher Docker image CI without needing downloading JAR files to this repository.
This PR closes:
  - #22 
  - d3b-center/clinical-data-flow#95

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
 <h2>Latest releases of the FHIR tools:</h2>
```
Comment (znatty22) [inline index.html:7]: We can get rid of this file now

**Review by znatty22 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (liberaliscomputing) [reply index.html:7]: So, the `index.html` is being updated with `tools.json` at https://github.com/FHIR/latest-ig-publisher. Then, after fetching upstream and rebasing upstream/master, we should manually remove `index.html` before pushing new commits. Do we want that?

Comment (znatty22) [reply index.html:7]: Oh I see, I didn't know it was still a part of the original repo. Let's leave it in

**Review by znatty22 — APPROVED** [review]
👍

**Pull request merged**